docs/ota-client-guide/modules/ROOT/pages/account-setup.adoc

```diff
@@ -8,24 +8,25 @@ We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}
 endif::[]
 
 
-If you do not want to mix up test and production software, you may use separate user logins or create additional environments.
+An environment is your working space in your https://connect.ota.here.com[OTA Connect account], where you can create and manage software update projects.
 
-== Separate user logins
+If you do not want to mix up test and production software xref:evaluation-to-prod.adoc#_integrate_ota_connect[during integration with OTA Connect], you may create additional environments. You can xref:ota-web::create-environment.adoc[create] up to 10 additional environments in one https://connect.ota.here.com[OTA Connect account].
 
-To manage the different stages of production workflow, you can create separate user logins:
+For example, you may need to have different environments for development, QA, and production.
```

Comment:
As it's an example for what is described in a previous couple of sentences, it would be nice to keep everything together in one paragraph.

Reply:
@HalynaDumych  it is actually a strong recommendation and not an example as it is not advisable to have one environment for testing and production. So instead I should modify it to become `To manage the different stages of production workflow, you should separate user logins`.

Reply:
Aha, got your point. But if we say "separate user logins", that may mean separate accounts. Maybe, it would be better to use "separate environments" instead?

Reply:
Sorry, that was a typo. I meant separate `separate environments` and not `user logins`.